Cargo.toml

```diff
@@ -39,6 +39,9 @@ serde_json = "1.0"
 sha2 = "0.9.1"
 structopt = "0.3"
 
+# FIXME: Support for authority key identifier just added, should be in 0.8.0 release
```

Comment:
NBD, but you can probably use 0.7 if you don't want to use a patch, since you're not doing authority key identifier parsing.

Reply:
I think the verification stuff doesn't exist in 0.7.